src/wkb2gltf.core.tests/ColorTests.cs

```diff
@@ -12,4 +12,12 @@ public void HexColorToRgbTest()
         var color = ColorTranslator.FromHtml(hexcolor);
         Assert.That(color.R == 255 && color.G == 85 && color.B == 85, Is.True);
     }
+    
+    [Test]
+    public void HexColorWithAlphaToRgbaTest()
+    {
+        var hexcolor = "#55ff5657";
```

Comment:
Invalid hex? https://www.color-hex.com/color/55ff5657

Reply:
that website doesn't support 4 channel hex codes.
you already have mentioned such case in [1] and also have test for it [2]
I will add section describing 4 channel hex in styling I guess because its not clear if alpha channel is first or last. The `ColorTranslator.ToHtml` expects alpha channel to be 1st, while in CSS it is on last position [3].
There is even issue about this in dotnet repo [4].

[1] https://github.com/Geodan/pg2b3dm/blob/245e0c7828c6c02734949d07394426ab8ec82887/styling.md?plain=1#L266
[2] https://github.com/Geodan/pg2b3dm/blob/245e0c7828c6c02734949d07394426ab8ec82887/src/wkb2gltf.core.tests/MaterialCreatorTests.cs#L17 
[3] https://developer.mozilla.org/en-US/docs/Web/CSS/hex-color
[4] https://github.com/dotnet/runtime/issues/27017